libmamba/CMakeLists.txt

```diff
@@ -505,7 +496,7 @@ macro(libmamba_create_target target_name linkage deps_linkage output_name)
         set_target_properties(
             ${target_name}
             PROPERTIES
-            PUBLIC_HEADER "${LIBMAMBA_PUBLIC_HEADERS}"
+            #PUBLIC_HEADER "${LIBMAMBA_PUBLIC_HEADERS}"
```

Comment:
Typo or should we remove?

Reply:
I want to keep it as a reminder that we should use this instead of a manual copy of the headers. The reason for not doing it is because we have subfolder under the include directory which is not handled correctly by cmake (it might be fixed in a recent version though).